packages/core/saved-objects/core-saved-objects-server-internal/src/routes/index.ts

```diff
@@ -49,6 +50,7 @@ export function registerRoutes({
   migratorPromise: Promise<IKibanaMigrator>;
   kibanaVersion: string;
   kibanaIndex: string;
+  typeRegistry: ISavedObjectTypeRegistry;
```

Comment:
This can be removed (I assume this was added before you decided to access the registry via the handler context - which was correct).

Reply:
good catch, thanks!